docs/roles/project-coordinator.rst

```diff
@@ -8,32 +8,43 @@ project lead for a specific project. This page explains more about the
 responsibilities and role of the project coordinator.
 
 
-Description
+Description 
 -----------
 
-The project coordinator is responsible for ensuring that all club projects have the resources needed to continue developing. This may include monetary resources, connections to mentors or the like and infrastructure to host the projects. 
+The project coordinator is responsible for ensuring that all club projects have
+the resources needed to continue developing.  This may include monetary
+resources, connections to mentors or the like and infrastructure to host the
+projects. 
 
 Eligibility requirements
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-The project coordinator must maintian good academic standing as defined by RIT to remain a member of eboard.
-Due to the hands on nature of the role, the project coordinator must be available in person during club and eboard meetings. If needed, a temporary replacement may be appointed to fill the role while the project coordinator is out on coop. In that case there should be good communication between the the two people in order to facilitate a smooth transition.
+The project coordinator must maintain good academic standing as defined by RIT
+to remain a member of eboard.  Due to the hands on nature of the role, the
+project coordinator must be available in person during club and eboard
+meetings.  If needed, a temporary replacement may be appointed to fill the role
+while the project coordinator is out on coop. In that case there should be good
```

Comment:
Co-op instead of coop.

Reply:
@axk4545 @jflory7 You guys wrote this, I'm going to defer to you.

Reply:
@Serubin Could you make this quick change with your PR?